supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/request/GetRequestHandler.scala

```diff
@@ -156,6 +156,16 @@ trait GetRequestHandler { this: RequestHandler =>
         ExecutionEvent("Complete in GCE / Cromwell Poll Interval", OffsetDateTime.parse(time.toString))
       }
 
+    val vmStartedEvent: Option[ExecutionEvent] = events.collectFirst {
+      case event if event.getWorkerAssigned != null =>
+        ExecutionEvent(CallMetadataKeys.VmStartTime, OffsetDateTime.parse(event.getTimestamp), Option.empty)
+    }
+
+    val vmEndedEvent: Option[ExecutionEvent] = events.collectFirst {
+      case event if event.getWorkerReleased != null =>
+        ExecutionEvent(CallMetadataKeys.VmEndTime, OffsetDateTime.parse(event.getTimestamp), Option.empty)
+    }
```

Comment:
Hm, do we have any tests of this behavior right now? I see a test above that's checking we can go from ExecutionEvents to cost info, but not one checking we go from special Google events to the right execution events.

Reply:
We do now!